vm/src/builtins/int.rs

```diff
@@ -701,6 +687,17 @@ impl PyInt {
     }
 }
 
+#[pyclass]
+impl PyRef<PyInt> {
+    #[pymethod(magic)]
+    fn int(self, vm: &VirtualMachine) -> PyRefExact<PyInt> {
+        self.into_exact_or(&vm.ctx, |zelf| unsafe {
+            // TODO: this is actually safe. we need better interface
```

Comment:
we should definitely start documenting all the unsafe better at some point.

Reply:
There's the allow-by-default [`undocumented_unsafe_blocks`](https://rust-lang.github.io/rust-clippy/master/index.html#undocumented_unsafe_blocks) lint, in case we want to activate that.

Reply:
That clippy option looks very good for us because we use a lot of unsafe blocks.

For this function, I'd like to make the basic PyRef creation functions to return PyRefExact first and turn them to PyRef later. In that case, everything will be safe by both semantics and rust grammars.